arviz/plots/kdeplot.py

```diff
@@ -289,6 +295,10 @@ def _fast_kde(x, cumulative=False, bw=4.5, xmin=None, xmax=None):
     std_x = entropy(x - xmin) * bw
 
     n_bins = min(int(len_x ** (1 / 3) * std_x * 2), n_points)
+    if n_bins < 2:
+        warnings.warn("kde plot failed, you may want to check your data")
```

Comment:
Duplication is not great, but if we make this a function then it doesn't save any lines of code